scripts/createYOEOLabelsFromTORSO-21.py

```diff
@@ -170,42 +148,38 @@ def range_limited_float_type_0_to_1(arg):
         continue
 
     name = os.path.splitext(img_name)[0]  # Remove file extension
-    imgwidth = frame['width']
-    imgheight = frame['height']
+    img_width = frame['width']
+    img_height = frame['height']
     annotations = []
 
     for annotation in frame['annotations']:
-        # Ignore if blurred or conceiled and should be ignored
-        if not ((args.ignore_blurred and annotation['blurred']) or
-            (args.ignore_conceiled and annotation['conceiled'])):
-
-            if annotation['type'] in CLASSES['bb_classes']:  # Handle bounding boxes
-                if annotation['in_image']:
-                    min_x = min(map(lambda x: x[0], annotation['vector']))
-                    max_x = max(map(lambda x: x[0], annotation['vector']))
-                    min_y = min(map(lambda x: x[1], annotation['vector']))
-                    max_y = max(map(lambda x: x[1], annotation['vector']))
-
-                    annowidth = max_x - min_x
-                    annoheight = max_y - min_y
-                    relannowidth = annowidth / imgwidth
-                    relannoheight = annoheight / imgheight
-
-                    center_x = min_x + (annowidth / 2)
-                    center_y = min_y + (annoheight / 2)
-                    relcenter_x = center_x / imgwidth
-                    relcenter_y = center_y / imgheight
-
-                    classID = CLASSES['bb_classes'].index(annotation['type'])  # Derive classID from index in predefined classes
-                    annotations.append(f"{classID} {relcenter_x} {relcenter_y} {relannowidth} {relannoheight}")  # Append to store it later
-                else:  # Annotation is not in image
-                    continue
-            elif annotation['type'] in CLASSES['segmentation_classes']:  # Handle segmentations
-                continue
-            elif annotation['type'] in CLASSES['ignored_classes']:  # Ignore this annotation
-                continue
-            else:
-                print(f"The annotation type '{annotation['type']}' is not supported or should be ignored. Image: '{img_name}'")
+        # Skip annotations, if is not a bounding box or should be skipped or is blurred or concealed and user chooses to skip them
+        if (annotation['type'] in CLASSES['segmentation_classes'] or  # Handled by segmentations
+            annotation['type'] in CLASSES['skip_classes'] or  # Skip this annotation class
+            args.skip_blurred and annotation['blurred'] or
```

Comment:
Is the operator precedence of `and` higher than `or`? Otherwise, brackets are missing. I would add they for readability ether way though.

Reply:
True. Will add them back